src/schedule.py

```diff
@@ -39,17 +39,17 @@ def AddEvent(self, weekday, event):
 EVENT_TYPES = {EVT_FIXED: 'FIXED', EVT_SUNSET: 'SUNSET', EVT_NEVER_ON: 'NEVER-ON'}
 
 
-def get_civil_twilight(city, event_date):
+def get_civil_twilight(observer, event_date, observer_timezone):

```

Comment:
Stay consistent with namingStyle.

Reply:
I've gotten used to camelCase for function names, and snake_case for args.  I don't know what's better or if anyone cares.

Reply:
okay. I'll fix it